KitchenLib/src/Customs/CustomGameDataObjects/CustomItem.cs

```diff
@@ -71,8 +82,8 @@ public override void Convert(GameData gameData, out GameDataObject gameDataObjec
             if (result.HoldPose != HoldPose) result.HoldPose = HoldPose;
             if (result.IsMergeableSide != IsMergeableSide) result.IsMergeableSide = IsMergeableSide;
 
-			//if (!string.IsNullOrEmpty(ColourBlindTag))
-			//ColorblindUtils.itemLabels.Add(new ItemLabel { itemId = result.ID, label = ColourBlindTag });
+			Main.LogDebug($"[CustomItem.Convert] [1.2] Convering Overrides");
```

Comment:
Typo - in all of the custom GDO files for this line

Reply:
Fixed this up